src/utils/safeHref.ts

```diff
@@ -21,33 +29,32 @@ export function configureSafeHref(configObject: SafeHrefConfigObj) {
     if (configObject.origin) {
         globalOrigin = configObject.origin
     }
+
+    if (configObject.additionalProtocols && configObject.additionalProtocols.length) {
+        safeProtocols.push(...configObject.additionalProtocols)
+    }
 }
 
 export function getUseSafeHref(): boolean {
     return useSafeHref
 }
 
-export function getURLInfo(url: string): URLInfo {
-    /**
-     * An array of the safely allowed url protocols
-     */
-    const safeProtocols = ['http:', 'https:', 'mailto:', 'tel:', 'data:']
-
+export function getURLInfo(url: string, allowProtocol: boolean): URLInfo {
     /**
      * - Find protocol of URL or set to 'relative'
      * - Find origin of URL
      * - Determine if sameOrigin
      * - Determine if protocol of URL is safe
      */
-    const protocolResult = url.match(PROTOCOL_REGEX)
+    const protocolResult = url.replace(' ', '').match(PROTOCOL_REGEX)
```

Comment:
Can't recall, will this only replace the first occurrence?

Reply:
Good catch here. Additionally I realized that we might want to check for other whitespace chars -- so updated this to be a little bit more robust